src/interface/src/styleguide/form.stories.ts

```diff
@@ -0,0 +1,113 @@
+import type { Meta, StoryObj } from '@storybook/angular';
+import { InputFieldComponent } from './input/input-field.component';
+import { Component } from '@angular/core';
+import { MatIconModule } from '@angular/material/icon';
+import { CommonModule } from '@angular/common';
+import { MatInputModule } from '@angular/material/input';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { InputDirective } from './input/input.directive';
+import { FormBuilder, ReactiveFormsModule, Validators } from '@angular/forms';
+import { ButtonComponent } from '@styleguide';
+import { MatCheckboxModule } from '@angular/material/checkbox';
+import { MatRadioModule } from '@angular/material/radio';
+
+@Component({
+  selector: 'sg-demo-form',
+  standalone: true,
+  imports: [
+    MatIconModule,
+    CommonModule,
+    MatInputModule,
+    MatFormFieldModule,
+    InputFieldComponent,
+    InputDirective,
+    ReactiveFormsModule,
+    ButtonComponent,
+    MatCheckboxModule,
+    MatRadioModule,
+  ],
+  styles: `
+    form {
+      display: flex;
+      gap: 48px;
+      text-align: left;
+    }
+    .flex {
+      display: flex;
+      gap: 16px;
+      flex-direction: column;
+      align-items: flex-start;
+      min-width: 200px;
+    }
+  `,
+  template: `
```

Comment:
is it possible to store this template as a separate file for maintainability, especially if there are other things we may want to add in the future?

Reply:
sure thing!